docs/bulletin.md

```diff
@@ -0,0 +1,74 @@
+# Bulletin
+
+## SLEAP v1.3.2
```

Comment:
I noticed you added the date in [shrivaths/changelog-announcement-2](https://github.com/talmolab/sleap/tree/shrivaths/changelog-announcement-2), but to keep changes organized, I would make any updates to the `bulletin.md` on this branch.

```suggestion
## SLEAP v1.3.2

_mm/dd/yyyy_

```